test/integration/targets/yum/tasks/yum.yml

```diff
@@ -232,3 +244,185 @@
     installroot: '/'
     state: removed
   register: yum_result
+
+- name: try to install not compatible arch rpm, should fail
+  yum:
+    name: "http://download.fedoraproject.org/pub/fedora/linux/releases/26/Server/armhfp/os/Packages/b/bash-4.4.12-5.fc26.armv7hl.rpm"
+    state: present
+  register: yum_result
+  ignore_errors: True
+
+- debug: var=yum_result
```

Comment:
Either remove the debug statements or add `verbosity` so they don't run by default but only when the verbosity gets cranked up.